cmd/gomtree/cmd/mutate.go

```diff
@@ -0,0 +1,181 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"math"
+	"os"
+	"slices"
```

Comment:
we're still accommodating go1.18, so perhaps we should use `golang.org/x/exp/slices` here?

Reply:
oh yes, sure. i'll make that change now.

Reply:
done

Reply:
could use slices due to generics. just replaced the variable to be a map